eoxserver/services/ows/wps/processes/get_height_profile.py

```diff
@@ -0,0 +1,211 @@
+# -----------------------------------------------------------------------------
+#
+# Project: EOxServer <http://eoxserver.org>
+# Authors: Martin Paces <martin.paces@eox.at>
+#          Mussab Abdalla<mussab.abdalla@eox.at>
+#
+# -----------------------------------------------------------------------------
+# Copyright (C) 2014 EOX IT Services GmbH
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies of this Software or works derived from this Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+# -----------------------------------------------------------------------------
+
+import csv
+from uuid import uuid4
+import os
+
+from math import radians, cos, sin, asin, sqrt
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from eoxserver.contrib import gdal
+
+from eoxserver.core import Component, implements
+
+from eoxserver.services.ows.wps.interfaces import ProcessInterface
+from eoxserver.services.ows.wps.parameters import (
+    LiteralData, ComplexData, FormatText, CDAsciiTextBuffer,
+    CDByteBuffer, FormatBinaryRaw, FormatBinaryBase64,
+)
+from eoxserver.services.ows.wps.exceptions import InvalidInputValueError
+
+from eoxserver.resources.coverages import models
+from eoxserver.backends.access import gdal_open
+
+
+class GetHeightProfileProcess(Component):
+    """ GetHeightProfileProcess defines a WPS process needed by the EOxC
+        DEM implementtation """
```

Comment:
I would not reference any other project here. A description of what the process does is sufficient.